test/scripts/cases/_olla.sh

```
@@ -0,0 +1,311 @@
+start_olla() {
+    local config_file="${1:-$TEST_CONFIG}"
+    local log_file="${2:-olla.log}"
+    
+    print_color "$YELLOW" "Starting Olla..."
+    
+    # Save current directory
+    local current_dir=$(pwd)
+    cd "$PROJECT_ROOT"
+    
+    # Check if port is available
+    local port="${TEST_PORT:-40114}"
```

Shellcheck (0.10.0)

[error] Literal carriage return. Run script through tr -d '\r' .

(SC1017)